[Doc Update][Extend a Teams personal tab across Microsoft 365] (#7596)
MicrosoftDocs/msteams-docs: msteams-platform/m365-apps/prerequisites.md

You need a Microsoft 365 developer subscription sandbox tenant to set up your dev environment. If you don't already have one, create a [sandbox tenant](/office/developer-program/microsoft-365-developer-program-get-started) or get a test tenant through your organization.

> > [!NOTE]
> > As you set up for testing, make sure to sign-in with the account you intend to use for testing and sign-out from any others.

Reviewer: ```suggestion
> [!NOTE]
> As you set up for testing, ensure that you sign out of all other accounts and sign in with the account you intend to use for testing.
```

To install Office 365 Beta Channel applications in your test environment:

1. Sign in to your test environment with your test tenant credentials.

> 1. Download and run the [Office Deployment Tool](https://www.microsoft.com/download/details.aspx?id=49117)
> 1. Select a local folder to store the extracted files

Reviewer: ```suggestion
1. Download and run the [Office Deployment Tool](https://www.microsoft.com/download/details.aspx?id=49117).
1. Select a local folder to store the extracted files.
```
